src/mixins/filing-template-mixin.ts

```diff
@@ -413,7 +413,6 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
       business: {
         legalType: this.getEntityType,
         identifier: this.getBusinessId,
-        legalName: this.getBusinessLegalName,
```

Comment:
Some of the other filings don't save legal name here in the business object, and if a NR is used then legal name is saved below on line 457, so I _hope_ this is OK in the back end.

I'm testing all 4 possibilities (full vs resto, numbered vs named) and so far it looks OK.